synapse/datamodel.py

```diff
@@ -718,7 +718,7 @@ def delForm(self, formname):
 
         form = self.forms.get(formname)
         if form is None:
-            raise s_exc.NoSuchForm(name=formname)
```

Comment:
This feels incorrect.  Perhaps it would be better to try/except in the one case where this is ok rather than silently failing in any other case? ( ie, dont change this, change the one caller who changed his expectation )

Reply:
This matches the behavior of addForm.